_field-types/dynamic.md

```diff
@@ -0,0 +1,342 @@
+        "patient": {
+          "properties": {
+            "id": {
+              "type": "text",
+              "fields": {
+                "keyword": {
+                  "type": "keyword",
+                  "ignore_above": 256
+                }
+              }
+            },
+            "name": {
+              "type": "text",
+              "fields": {
+                "keyword": {
+                  "type": "keyword",
+                  "ignore_above": 256
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+---
+
+## Example: Create index with dynamic parameter set to `false`
+
+1. Create an index with `dynamic` set to `false` and explicit mappings using the following request:
+
+```json
+PUT testindex1
+{
+  "mappings": {
+    "dynamic": false,
+    "properties": {
+      "patient": {
+        "properties": {
+          "id": {
+            "type": "keyword"
+          },
+          "name": {
+            "type": "keyword"
+          }
+        }
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+2. Index a document with an object field patient containing two string fields and additional unmapped fields using the following request:
+
+```json
+PUT testindex1/_doc/1
+{ 
+  "patient": { 
+    "name" : "John Doe",
+    "id" : "123456"
+  },
+  "room": "room1",
+  "floor": "1"
+}
+```
+{% include copy-curl.html %}
+
+3. Check the mapping using the following request:
+
+```json
+GET testindex1/_mapping
+```
+{% include copy-curl.html %}
+
+The following response shows that the new fields `room` and `floor` were not added to the mapping, which remained unchanged:
+
+```json
+{
+  "testindex1": {
+    "mappings": {
+      "dynamic": "false",
+      "properties": {
+        "patient": {
+          "properties": {
+            "id": {
+              "type": "keyword"
+            },
+            "name": {
+              "type": "keyword"
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+4. Get the unmapped fields `room` and `floor` from the document using the following request:
+
+```json
+PUT testindex1/_doc/1
+{ 
+  "patient": { 
+    "name" : "John Doe",
+    "id" : "123456"
+  },
+  "room": "room1",
+  "floor": "1"
+}
+```
+
+The following request searches the fields `room` or `floor`, with a response confirming no results returned:
+
+```json
+POST testindex1/_search
+{
+  "query": {
+    "term": {
+      "room": "room1"
+    }
+  }
+}
+```
+
+The following response confirms no returned results:
+
+```json
+{
+  "took": 3,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 0,
+      "relation": "eq"
+    },
+    "max_score": null,
+    "hits": []
+  }
+}
+```
+
+---
+
+## Example: Create an index with `dynamic` set to `strict`
+
+1. Create an index with `dynamic` set to `strict` and explicit mappings using the following request:
```

Comment:
Can we reverse the order here?: "Create an index with explicit mappings and `dynamic` set to `strict`..."